bundles/CoreBundle/DependencyInjection/Compiler/AreabrickPass.php

```diff
@@ -204,7 +204,7 @@ protected function handleContainerAwareDefinition(ContainerBuilder $container, D
     protected function findBundleBricks(ContainerBuilder $container, string $name, array $metadata, array $excludedClasses = []): array
     {
         $sourcePath = is_dir($metadata['path'].'/src') ? $metadata['path'].'/src' : $metadata['path'];
-        $directory = $sourcePath.'/Document/Areabrick';
+        $directory = $sourcePath.DIRECTORY_SEPARATOR.'Document'.DIRECTORY_SEPARATOR.'Areabrick';
```

Comment:
should we consider it in the source path as well?

Reply:
@dvesh3 `$sourcePath` is removed with str_replace bellow on line 230 so I don't think it is necessary